src/components/organisms/NavBarSchedule/NavBarSchedule.scss

```diff
@@ -126,3 +134,28 @@ $schedule-btn--height: 51px;
     padding: $spacing--sm 0px;
   }
 }
+
+@media screen and (max-width: $media-breakpoint--md) {
+  .NavBarSchedule {
+    width: 100%;
+  }
+
+  .NavBarWrapper {
+    &__download-buttons {
+      width: 100%;
+    }
+
+    &__close-button {
+      height: $close-btn-height;
+      width: $close-btn-width;
+      display: block;
+      position: absolute;
+      right: $spacing--md;
+      z-index: z(navbar-schedule-download);
+
```

Comment:
Let's place this close button closer to the navbar. Just under the days.

![image](https://user-images.githubusercontent.com/63313732/132701232-23781c10-55a7-4813-96ea-8d970575521e.png)